test/catalog.go

```diff
@@ -51,6 +51,33 @@ func (c *Catalog) SecretWithoutAnnotation(name string, ns string) *corev1.Secret
 	}
 }
 
+// BuildWithClusterBuildStrategyAndSourceAnnotation gives you an specific Build CRD
+func (c *Catalog) BuildWithClusterBuildStrategyAndSourceAnnotation(name string, ns string, strategyName string, secretName string) *build.Build {
+	buildStrategy := build.ClusterBuildStrategyKind
+	return &build.Build{
+		ObjectMeta: metav1.ObjectMeta{
```

Comment:
you have an strange indentation here, please double check

Reply:
Done!